src/libs/ValidationUtils.ts

```diff
@@ -78,33 +69,27 @@ function isValidPastDate(date) {
 
 /**
  * Used to validate a value that is "required".
- *
- * @param {*} value
- * @returns {Boolean}
  */
-function isRequiredFulfilled(value) {
-    if (_.isString(value)) {
-        return !_.isEmpty(value.trim());
+function isRequiredFulfilled(value: string | Date | unknown[] | Record<string, unknown>): boolean {
+    if (typeof value === 'string') {
+        return value.trim().length > 0;
     }
-    if (_.isDate(value)) {
+
+    if (isDate(value)) {
         return isValidDate(value);
     }
-    if (_.isArray(value) || _.isObject(value)) {
-        return !_.isEmpty(value);
+    if (Array.isArray(value) || isObject(value)) {
```

Comment:
Same here

```suggestion
    if (Array.isArray(value) || typeof value === 'object') {
```

Reply:
Yeah, that's true, but I thought those other types were already narrowed with the other `if` statements above. But okay, not a big deal, let's leave the way it is 👍🏻